src/Data/List/Relation/Unary/Grouped/Properties.agda

```diff
@@ -0,0 +1,76 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Property related to Grouped
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.List.Relation.Unary.Grouped.Properties where
+
+open import Data.Bool using (true; false)
+open import Data.List
+open import Data.List.Relation.Unary.All as All using (All; []; _∷_)
+import Data.List.Relation.Unary.All.Properties as All
+open import Data.List.Relation.Unary.AllPairs as AllPairs using (AllPairs; []; _∷_)
+open import Data.List.Relation.Unary.Grouped
+open import Function.Base using (_∘_)
+open import Function.Equality using (_⟨$⟩_)
+open import Function.Equivalence using (Equivalence; _⇔_)
+open import Relation.Binary as B using (REL; Rel)
+open import Relation.Unary as U using (Pred)
+open import Relation.Nullary using (¬_; does; yes; no)
+open import Relation.Nullary.Negation using (contradiction)
+open import Level
+
+private
+  variable
+    a b c p q : Level
+    A : Set a
+    B : Set b
+    C : Set p
+
+------------------------------------------------------------------------
+-- map
+
+module _ (P : Rel A p) (Q : Rel B q) where
+
+  map⁺ : ∀ {f xs} → (∀ {x y} → P x y ⇔ Q (f x) (f y)) → Grouped P xs → Grouped Q (map f xs)
```

Comment:
Please could you use the new `_⇔_` from `Function` (originally in `Function.Bundles`)?